addOns/ascanrules/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## [55] - 2023-06-06
 ### Changed
 - The Parameter Tamper Scan rule now includes example alert functionality for documentation generation purposes (Issue 6119)
+- Moved Timing utils to commonlib
```

Comment:
- Note that the entries should be added under the `Unreleased` version. This one is old/released already.
- Use `Depend on newer version of Common Library add-on.` instead.